cumulusci/tasks/salesforce/UninstallPackagedIncremental.py

```diff
@@ -36,7 +40,10 @@ def _init_options(self, kwargs):
         self.options["purge_on_delete"] = process_bool_arg(
             self.options.get("purge_on_delete", True)
         )
-        self.options["ignore"] = self.options.get("ignore") or {}
+        self.options["ignore"] = self.options.get("ignore", {})
```

Comment:
I prefer `self.options.get("ignore") or {}` because it handles the case where the key was explicitly set to None.

Reply:
👨‍🎓 👍